kernel/nativevalues.ml

```diff
@@ -786,6 +790,79 @@ let next_down accu x =
   if is_float x then no_check_next_down x
   else apply accu x
 
+(** Support for primitive strings *)
```

Comment:
Please keep that in its original file, so as to make the diff more readable.

Reply:
I'm sorry, I don't understand that comment.

Reply:
You move the code below from another file, making your edits harder to spot, please keep it there.

Reply:
I don't think I moved anything, this seems like only additions.

The only place where I moved things is in the syntax plugin, but I made that a separate commit.

Reply:
That's what I mean. Well I guess isolating it like that in a commit make it somewhat fine.